### PR TITLE
docs: Update props JSDOC (A-K) for Gamut-Labs

### PR DESCRIPTION
**Pull request by dreamwasp**
### PR Checklist

Adds better descriptions to Storybooks component props + fixes a couple small bugs along the way.
- [ ] Related to designs:
- [x] Related to JIRA ticket: [GM-47](https://codecademy.atlassian.net/browse/GM-47)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.41b5d6.0<br/>@codecademy/styleguide@21.24.2-alpha.41b5d6.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.e87797.0<br/>@codecademy/styleguide@21.24.2-alpha.e87797.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601300791768fe356f592f35--gamut-preview.netlify.app>https://601300791768fe356f592f35--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601300791768fe356f592f35>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.7e757d.0<br/>@codecademy/styleguide@21.24.2-alpha.7e757d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6013025cebf8ff20b9cb255d--gamut-preview.netlify.app>https://6013025cebf8ff20b9cb255d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6013025cebf8ff20b9cb255d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.f2c5c6.0<br/>@codecademy/styleguide@21.24.2-alpha.f2c5c6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601317cee9d64680caffecda--gamut-preview.netlify.app>https://601317cee9d64680caffecda--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601317cee9d64680caffecda>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.696081.0<br/>@codecademy/styleguide@21.24.2-alpha.696081.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60132e38118a516ed979d094--gamut-preview.netlify.app>https://60132e38118a516ed979d094--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60132e38118a516ed979d094>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.95c8a6.0<br/>@codecademy/styleguide@21.24.2-alpha.95c8a6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601333828d2aa0009143d371--gamut-preview.netlify.app>https://601333828d2aa0009143d371--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601333828d2aa0009143d371>Deploy Logs</a></p>

File: packages/gamut-labs/src/brand/Avatar/index.tsx [hunk 1]
```diff
@@ -12,7 +12,11 @@ type AvatarImageProps =
   | { alt?: never; 'aria-labelledby': string };
 
 type AvatarBaseProps = {
+  /**path to image asset
+   */
```
Comment (codecaaron) [inline packages/gamut-labs/src/brand/Avatar/index.tsx:16]: Just a little comment formattting nit.
```suggestion
  /**
   * path to image asset
   */
```

**Review by codecaaron — COMMENTED** [review]
Noice!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.1-alpha.7eb269.0<br/>@codecademy/styleguide@21.24.2-alpha.7eb269.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60133abf362e6b008d4a7f23--gamut-preview.netlify.app>https://60133abf362e6b008d4a7f23--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60133abf362e6b008d4a7f23>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.16.5-alpha.7e486a.0<br/>@codecademy/styleguide@22.0.4-alpha.7e486a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60133f274bee210f991c413d--gamut-preview.netlify.app>https://60133f274bee210f991c413d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60133f274bee210f991c413d>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
Love it! 💯

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@8.0.1-alpha.24b75f.0<br/>@codecademy/styleguide@23.0.1-alpha.24b75f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6014396ee47a113b281c88c1--gamut-preview.netlify.app>https://6014396ee47a113b281c88c1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6014396ee47a113b281c88c1>Deploy Logs</a></p>

**Pull request merged**